src/components/homePage.tsx

```diff
@@ -0,0 +1,9 @@
+export const HomePage = () => {
```

Comment:
Consider renaming the file to 'HomePage.tsx' to match the primary named export 'HomePage'.